### PR TITLE
feat(election-manager): add Adjudicate buttons to write-ins screen

### PR DESCRIPTION
**Pull request by taravancil**
## Overview
#1782 

This implements the "Adjudicate {n} write-ins for {contest}" buttons on the "Write-ins" tab in election manager. It does not wire up the buttons to open the full transcription screen yet, that will come in the next PR.

Buttons are disabled when CVRs have not been imported, or when a given contest does not have any write-ins. 

## Demo Video or Screenshot
localhost_3000_write-ins (1)
localhost_3000_write-ins (2)

## Testing Plan 
Manual testing for now, will add comprehensive tests as this feature progresses out of its rough prototype state. Feels counterproductive to do that now.

## Checklist
- [x] I have added [logging](https://github.com/votingworks/vxsuite/tree/main/libs/logging) where appropriate to any new user actions, system updates such as file reads or storage writes, or errors introduced.
- [x] I have added JSDoc comments to any newly introduced exports

- [x] I have added a screenshot and/or video to this PR to demo the change
- [ ] I have added the "user_facing_change" label to this PR to automate an announcement in #machine-product-updates

### CODE REVIEW

File: frontends/election-manager/src/screens/write_ins_screen.tsx
Comment (taravancil) [inline frontends/election-manager/src/screens/write_ins_screen.tsx:35]: @eventualbuddha for the sake of trying to prototype this UI quickly, I opted to save the work of rethinking how we represent write in values for a separate ticket #1793.

File: frontends/election-manager/src/components/write_ins_transcription_screen.tsx
Comment (beausmith) [reply frontends/election-manager/src/components/write_ins_transcription_screen.tsx:16]: Meh, it's arguably a "screen". Your call.

**Review by beausmith — APPROVED** [review]
Looks good!

File: frontends/election-manager/src/screens/write_ins_screen.tsx
Comment (beausmith) [inline frontends/election-manager/src/screens/write_ins_screen.tsx:68]: We are using UTF-8 so you can use real smart quotes!
```suggestion
                for “{contest.section} {contest.title}”
```

Comment (taravancil) [reply frontends/election-manager/src/screens/write_ins_screen.tsx:68]: Nice! Going for it

File: frontends/election-manager/src/screens/write_ins_screen.tsx
Comment (beausmith) [inline frontends/election-manager/src/screens/write_ins_screen.tsx:75]: Primaries! Good call putting the party abbreviation here.

Comment (taravancil) [reply frontends/election-manager/src/screens/write_ins_screen.tsx:75]: Yeah! I just happened to test with a primary election and was like, "oh, right, primaries are a thing"

**Comment by taravancil** [comment]
@eventualbuddha I'm going to merge this, but if you want to see any changes feel free to add your review and I will make follow up tickets!

File: frontends/election-manager/src/screens/write_ins_screen.tsx
Comment (eventualbuddha) [inline frontends/election-manager/src/screens/write_ins_screen.tsx:26]: This should have type parameters, otherwise the `key` and `value` are both `any`:

```suggestion
  const writeInCountsByContest = new Map<ContestId, number>();
```

File: frontends/election-manager/src/screens/write_ins_screen.tsx
Comment (eventualbuddha) [inline frontends/election-manager/src/screens/write_ins_screen.tsx:32]: More specifically:
```suggestion
          const votes = cvr[contestId] as ContestOptionId[];
```

File: frontends/election-manager/src/screens/write_ins_screen.tsx
Comment (eventualbuddha) [inline frontends/election-manager/src/screens/write_ins_screen.tsx:33]: More broad suggestion:
```suggestion
      for (const [key, value] of Object.entries(cvr)) {
        if (Array.isArray(value)) {
          const contestId = key;
          const votes = value as ContestOptionId[];
          // TODO: this does not capture all the ways we represent write-ins, must be
```

**Pull request merged**
